modules/Microsoft.Compute/galleries/images/readme.md

```diff
@@ -39,7 +39,8 @@ This module deploys an Image Definition in a Shared Image Gallery.
 | `eula` | string | `''` |  | The Eula agreement for the gallery Image Definition. Has to be a valid URL. |
 | `excludedDiskTypes` | array | `[]` |  | List of the excluded disk types. E.g. Standard_LRS. |
 | `hyperVGeneration` | string | `'V1'` | `[V1, V2]` | The hypervisor generation of the Virtual Machine. Applicable to OS disks only. - V1 or V2. |
-| `imageDefinitionDescription` | string | `''` |  | The description of this gallery Image Definition resource. This property is updatable. |
+| `securityType` | string | | `[TrustedLaunch, ConfidentialVM, ConfidentialVMSupported]` | Security type refers to the different security features available for a Generation 2 virtual machines. Requires V2 hyperVGeneration. |
```

Comment:
As a last step, I'd suggest to regenerate the readme through the Set-ModuleReadme utility, so that also the API version update will be reflected.